Library/Source/RuntimeImpl.cpp

```diff
@@ -59,6 +65,23 @@ namespace Babylon
         });
     }
 
+    void RuntimeImpl::UpdateWindow(float width, float height, void* nativeWindowPtr)
```

Comment:
Is this a new direct dependency? I'm not sure we want to have this file (or what it will eventually turn into) depend on bgfx.

Reply:
yes, new contract function for Android.